.github/workflows/docs.yml

```
@@ -0,0 +1,49 @@
+        id: pnpm-install
+        with:
+          version: 8
+          run_install: false
+      - name: Get pnpm version
+        id: pnpm-version
+        run: |
+          echo "$(pnpm --version)"
+
+      - name: Get pnpm store directory
+        id: pnpm-cache
+        run: |
```

actionlint (1.7.3)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)